### ISSUE TITLE
unsupported protocol scheme "wakeup"

### ISSUE THREAD

**Issue by rkeene** [zankich/groku#9]
```
$ ./groku discover   
Get wakeup: MAC=b8:3e:59:39:28:10;Timeout=10query/device-info: unsupported protocol scheme "wakeup"
$ ./groku list    
Get wakeup: MAC=dc:3a:5e:17:97:99;Timeout=10query/device-info: unsupported protocol scheme "wakeup"
$ 
```

Adding `fmt.Println("Location = ", location);` after groko.go:279 produces `Location =  WAKEUP: MAC=b8:3e:59:39:28:10;Timeout=10`

So the service discovery isn't returning a Location header that's expected, apparently.

**Comment by zankich** [comment]
@rkeene It looks like they may have changed the response format from SSDP. Unfortunately I no longer have a roku, so I can't test this out with the latest firmware. 

The raw [response](https://github.com/zankich/groku/blob/f766cae068fab923f291f0bab587cbff7ac05ed0/groku.go#L192), should contain the address of the roku, so if you inspect that output, that should help you determine where the address is now.

**Comment by zankich** [comment]
hey @rkeene i finally got a new roku and fixed this issue in groku [0.4.1](https://github.com/zankich/groku/releases/tag/0.4.1)
